tests/utils/test_configdict.py

```diff
@@ -0,0 +1,182 @@
+import pytest
+
+from impactlab_tools.utils.configdict import ConfigDict, gather_configtree
+
+
+@pytest.fixture
+def simple_nested_tree():
+    return {'a': 1, 'b': {'a': 2}, 'c': 3, 'd-4': 4, 'e_5': 5, 'F': 6}
+
+
+def test_configdict_climbs_tree(simple_nested_tree):
+    conf = ConfigDict(simple_nested_tree)
+    assert conf['c'] == 3
```

Comment:
This doesn't check if it climbs the tree (as from the test function name). Do you want `conf['b']['c'] == 3`?

Reply:
Thanks for that, @jrising . Fixed in new e615021.